x-pack/test/send_search_to_background_integration/tests/apps/dashboard/async_search/send_to_background_relative_time.ts

```diff
@@ -0,0 +1,120 @@
+  const testSubjects = getService('testSubjects');
+  const log = getService('log');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects([
+    'common',
+    'header',
+    'dashboard',
+    'visChart',
+    'home',
+    'timePicker',
+  ]);
+  const dashboardPanelActions = getService('dashboardPanelActions');
+  const inspector = getService('inspector');
+  const pieChart = getService('pieChart');
+  const find = getService('find');
+  const dashboardExpect = getService('dashboardExpect');
+  const queryBar = getService('queryBar');
+  const browser = getService('browser');
+  const sendToBackground = getService('sendToBackground');
+
+  describe('send to background with relative time', () => {
+    before(async () => {
+      await PageObjects.common.sleep(5000);
+      await PageObjects.common.navigateToUrl('home', '/tutorial_directory/sampleData', {
+        useActualUrl: true,
+      });
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      // use sample data set because it has recent relative time range and bunch of different visualizations
+      await PageObjects.home.addSampleDataSet('flights');
+      await retry.tryForTime(10000, async () => {
+        const isInstalled = await PageObjects.home.isSampleDataSetInstalled('flights');
+        expect(isInstalled).to.be(true);
+      });
+      await PageObjects.common.navigateToApp('dashboard');
+    });
+
+    after(async () => {
+      await PageObjects.common.navigateToUrl('home', '/tutorial_directory/sampleData', {
+        useActualUrl: true,
+      });
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.home.removeSampleDataSet('flights');
+      const isInstalled = await PageObjects.home.isSampleDataSetInstalled('flights');
+      expect(isInstalled).to.be(false);
+    });
+
+    it('Saves and restores a session with relative time ranges', async () => {
+      await PageObjects.dashboard.loadSavedDashboard('[Flights] Global Flight Dashboard');
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.timePicker.pauseAutoRefresh(); // sample data has auto-refresh on
+      // await PageObjects.timePicker.setCommonlyUsedTime('sample_data range');
+      await queryBar.submitQuery();
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.dashboard.waitForRenderComplete();
+      await checkSampleDashboardLoaded();
+
+      await sendToBackground.expectState('completed');
+      await sendToBackground.save();
+      await sendToBackground.expectState('backgroundCompleted');
+      const savedSessionId = await dashboardPanelActions.getSearchSessionIdByTitle(
+        '[Flights] Airline Carrier'
+      );
+      const resolvedTimeRange = await getResolvedTimeRangeFromPanel('[Flights] Airline Carrier');
+
+      // load URL to restore a saved session
+      const url = await browser.getCurrentUrl();
```

Comment:
Once management is available, we should replace this part of the test with real navigation  @tsullivan

Reply:
Also, @Dosant did you create an issue to restore state from the Saved Object if a session ID is present in the URL?
If so, what's its number?

Reply:
Just created: https://github.com/elastic/kibana/issues/87446